### PR TITLE
add points for jupyter

### PR DESCRIPTION
**Pull request by pkking**
This pr related with https://github.com/opensourceways/xihe-statistics/pull/13
1. send the cloud event for points
2. we remove the operatelog msg sent by cloud into `operate_log` topic, instead we just subscribe the `cloud` topic for operatelog stuff.

### CODE REVIEW

File: common/domain/message/message.go [hunk 1]
```diff
@@ -8,8 +8,8 @@ type MsgNormal struct {
 	Type      string            `json:"type"`
 	User      string            `json:"user"`
 	Desc      string            `json:"desc"`
-	Details   map[string]string `json:"details"`
-	CreatedAt int64             `json:"created_at"`
+	Details   map[string]string `json:"info"`
```
Comment (zengchen1024) [inline common/domain/message/message.go:11]: why did you change the tag

File: config/config.go [hunk 2]
```diff
@@ -82,6 +83,7 @@ type Config struct {
 	MQ          kafka.Config         `json:"mq"           required:"true"`
 	MQTopics    messages.Topics      `json:"mq_topics"    required:"true"`
 	Points      pointsConfig         `json:"points"`
+	Cloud       cloudmsg.Config      `json:"cloud"        required:"true"`
```
Comment (zengchen1024) [inline config/config.go:86]: add the config item in function of configItems

File: cloud/domain/service/service.go [hunk 3]
```diff
@@ -68,9 +68,6 @@ func (r *CloudService) SubscribeCloud(
 		return
 	}
 
-	// add operate log
-	r.sender.AddOperateLogForCloudSubscribe(u, p.CloudId)
```
Comment (zengchen1024) [inline cloud/domain/service/service.go:72]: it should delete the https://github.com/opensourceways/xihe-server/blob/dev/infrastructure/messages/sender.go#L221

File: cloud/infrastructure/message/cloud.go [hunk 4]
```diff
@@ -0,0 +1,52 @@
+package message
```
Comment (zengchen1024) [inline cloud/infrastructure/message/cloud.go:1]: change the directory of 'cloud/infrastructure/message' to 'cloud/infrastructure/messageadapter'

File: cloud/infrastructure/message/cloud.go [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+type publisher struct {
+	cfg       Config
+	publisher common.Publisher
+}
+
+func (s publisher) SubscribeCloud(m *message.MsgCloudConf) error {
+	msg := CloudCreateMsg{
+		MsgNormal: common.MsgNormal{
+			Type:      s.cfg.JupyterCreated.Name,
+			User:      m.User,
```
Comment (zengchen1024) [inline cloud/infrastructure/message/cloud.go:34]: it is better to set the Desc of MsgNormal

File: cloud/infrastructure/message/cloud.go [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+			CreatedAt: utils.Now(),
+		},
+		PodId:     m.PodId,
+		CloudId:   m.CloudId,
+		CloudName: m.CloudName,
+	}
+	return s.publisher.Publish(s.cfg.JupyterCreated.Topic, msg, nil)
```
Comment (zengchen1024) [inline cloud/infrastructure/message/cloud.go:41]: leave a blank line before return

File: cloud/infrastructure/message/cloud.go [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+func (s publisher) ReleasePod(m *message.MsgPod) error {
+	// no impl for now
+	return nil
```
Comment (zengchen1024) [inline cloud/infrastructure/message/cloud.go:46]: why don't implement it

File: cloud/infrastructure/message/config.go [hunk 8]
```diff
@@ -0,0 +1,10 @@
+package message
+
+type Topics struct {
+	CloudCreate topicConfig `json:"cloud_create"`
```
Comment (zengchen1024) [inline cloud/infrastructure/message/config.go:4]: Why not define it at https://github.com/opensourceways/xihe-server/pull/541/files#diff-da9bf2b219bdbf2997193cbcb1604de368abce68cd478a50f6afc597206568f4R51

**Review by zengchen1024 — CHANGES_REQUESTED** [review]

File: common/domain/message/message.go (hunk 1 above)
Comment (pkking) [reply common/domain/message/message.go]: > why did you change the tag

fixed

File: config/config.go (hunk 2 above)
Comment (pkking) [reply config/config.go:86]: > add the config item in function of configItems

done

File: cloud/domain/service/service.go (hunk 3 above)
Comment (pkking) [reply cloud/domain/service/service.go:72]: > it should delete the https://github.com/opensourceways/xihe-server/blob/dev/infrastructure/messages/sender.go#L221

done

File: cloud/infrastructure/message/cloud.go (hunk 4 above)
Comment (pkking) [reply cloud/infrastructure/message/cloud.go]: > change the directory of 'cloud/infrastructure/message' to 'cloud/infrastructure/messageadapter'

done

File: cloud/infrastructure/message/cloud.go (hunk 5 above)
Comment (pkking) [reply cloud/infrastructure/message/cloud.go]: > it is better to set the Desc of MsgNormal

done

File: cloud/infrastructure/message/cloud.go (hunk 6 above)
Comment (pkking) [reply cloud/infrastructure/message/cloud.go]: > leave a blank line before return

done

File: cloud/infrastructure/message/config.go (hunk 8 above)
Comment (pkking) [reply cloud/infrastructure/message/config.go]: > Why not define it at https://github.com/opensourceways/xihe-server/pull/541/files#diff-da9bf2b219bdbf2997193cbcb1604de368abce68cd478a50f6afc597206568f4R51

done
